editions/tw5.com/tiddlers/features/DateFormat.tid

```diff
@@ -4,7 +4,7 @@ tags: Features
 title: DateFormat
 type: text/vnd.tiddlywiki
 
-The ViewWidget accepts a `template` attribute that allows the format of date values to be specified. The format string is processed with the following substitutions:
+If the ViewWidget has a `format=date` attribute, the date can be formatted using the `template` attribute. Specify this format by setting `template="..."` where `...` is a combinaton of the following substitution strings:
```

Comment:
I'm not sure, if this is an improvement.

Reply:
There is no "if" ... The description tiddler should tell about facts. ... Examples can start with "If"